src/creatures/players/player.cpp

```diff
@@ -5200,6 +5219,129 @@ void Player::setTibiaCoins(int32_t v) {
 	coinBalance = v;
 }
 
+int32_t Player::getCleavePercent(bool useCharges) const {
+	int32_t result = cleavePercent;
+	for (const auto item : getEquippedItems()) {
+		const ItemType &it = Item::items[item->getID()];
+		const int32_t &cleave_percent = it.abilities->cleavePercent;
```

Comment:
crashing here (when you open the cyclopedia > combat stats window), you need to do the "if (it.abilities)" check before "const int32_t &cleave_percent = it.abilities->cleavePercent;"

the same applies to the functions below.

Reply:
how i solved it:
```cpp
const ItemType &it = Item::items[item->getID()];
if (it.abilities) {
  const int32_t &cleave_percent = it.abilities->cleavePercent;
  if (cleave_percent != 0) {
    result += cleave_percent;
    uint16_t charges = item->getCharges();
    if (useCharges && charges != 0) {
      g_game().transformItem(item, item->getID(), charges - 1);
    }
  }
}
```

Reply:
Fixed, thanks.